app/controllers/provider_orders_controller.rb

```diff
@@ -27,6 +27,7 @@ def destroy
     if result.success?
       flash[:success] = "#{'Item'.pluralize(catalog_items.count)} successfully #{method}"
     else
+      Rails.logger.error("#{method.capitalize} Provider Order Error: #{result.inspect}")
```

Comment:
likewise here?

Reply:
same answer, this is an `echo_client` response